### PR TITLE
[MM-36313] - Fix hardcoded value for about modal border

### PR DESCRIPTION
**Pull request by nevyangelova**
#### Summary
This PR fixes a hardcoded value for the about modal border colour so that the colour can update when we are using different themes. 

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-36313

#### Release Note

-->
```release-note: NONE

```

### CODE REVIEW

**Review by jgilliam17 — APPROVED** [review]
Thanks @nevyangelova 
Tested, looks good to merge.
- Verified about modal border.

File: sass/routes/_about-modal.scss [hunk 1]
```diff
@@ -14,7 +14,7 @@
                 align-items: center;
                 padding: 20px 25px;
                 border: none;
-                border-bottom: 1px solid #ddd;
+                border-bottom: 1px solid var(--center-channel-color-08);
```
Comment (hmhealey) [inline sass/routes/_about-modal.scss:17]: Can you use the rgba version of the theme colour here?

**Review by hmhealey — CHANGES_REQUESTED** [review]

File: sass/routes/_about-modal.scss (hunk 1 above)
Comment (michelengelen) [reply sass/routes/_about-modal.scss:17]: what @hmhealey said! :D

**Review by michelengelen — APPROVED** [review]
only the change request from @hmhealey ... nothing more to find here! ✨

**Review by hmhealey — APPROVED** [review]

**Pull request merged**